install/install.php

```diff
@@ -87,20 +87,32 @@ public function wp_version_notice() {
 	 * @return array
 	 */
 	public static function get_default_options() {
+		$show_option = defined( 'PLL_SHOW_LANGUAGE_FROM_CONTENT_OPTION' ) && PLL_SHOW_LANGUAGE_FROM_CONTENT_OPTION;
+		/**
+		 * Filters whether the "The language is set from content" option should be shown on new installations.
+		 * This is only available during Polylang's first activation.
+		 *
+		 * @since 3.7
```

Comment:
FYI, as there is no milestone 3.7 now and the issue is in 3.6 backlog, I moved the PR in 3.6 project